pkg/conduit/config.go

```diff
@@ -29,8 +30,8 @@ const (
 	DBTypePostgres = "postgres"
 	DBTypeInMemory = "inmemory"
 
-	SchemaTypeConfluent = "confluent"
-	SchemaTypeInMemory  = "inmemory"
+	SchemaRegistryTypeConfluent = "confluent"
+	SchemaRegistryTypeBuiltin   = "builtin"
```

Comment:
Hmm, thinking out loud - I changed the `inmemory` option to `builtin`, since it won't necessarily be in-memory, the goal is to use the same persistence layer as Conduit for storing schemas. Not I'm thinking if the other one should be called `standalone` (to mimic the terminology in plugins) or `external` (as the opposite of "builtin")? 🤔 cc @hariso @maha-hajja

Reply:
`builtin` sounds good to me. `confluent` is still a better option to me, because it makes it clear what kind of schema registry is supported.

Reply:
Alright, I'll leave it as it is, I also like that `confluent` makes it explicit what API is expected.